scripts/src/javatests/org/oppia/android/scripts/coverage/RunCoverageTest.kt

```diff
@@ -317,7 +317,13 @@ class RunCoverageTest {
 
     val expectedResult = getExpectedHtmlText(kotlinFilePath)
 
-    assertThat(readHtmlReport(kotlinFilePath)).isEqualTo(expectedResult)
+    val unescapedHtmlReport = readHtmlReport(kotlinFilePath)
```

Comment:
Could you maybe use Guava to unescape the HTML here & for other tests in the PR?

Reply:
@BenHenning, Uhm... I'm not sure if Guava has methods to unescape the same?

[Guava Doc](https://guava.dev/releases/19.0/api/docs/com/google/common/escape/Escaper.html#escape(java.lang.String):~:text=Function%3CString,given%20literal%20string.)  |  [stackoverflow thread](https://stackoverflow.com/a/34091458)

or does it? or is there any other better way to do it than direct conversion.

Reply:
Good call, I suppose this approach seems reasonable then. I'd rather not introduce another dependency if we can avoid it, even for tests.